### ISSUE TITLE
webpacker:install leads to a bin/yarn conflict

### ISSUE THREAD

**Issue by rubys** [rails/webpacker#278]
### Steps to reproduce

```
rails new wpinstall
cd wpinstall/
echo "gem 'webpacker', github: 'rails/webpacker'" >> Gemfile
bundle install
rails webpacker:install
```

### Expected behavior
On a clean repository, this should complete without conflict.

### Actual behavior
```
conflict  bin/yarn
Overwrite /home/rubys/git/awdwr/edition4/work-51/depot/bin/yarn? (enter "h" for help) [Ynaqdh] Y
       force  bin/yarn
```

### System configuration
**Rails version**: 5.1.0.rc2

**Ruby version**: 2.4.0

**Comment by ytbryan** [comment]
Yup @y-yagi fixed it. This can be closed. Thanks for reporting, rubys

**Comment by matthewd** [comment]
webpacker should be more explicit about only creating that file if it doesn't exist, so it's only supplying it for pre-5.1 Rails apps. Otherwise we're going to have ongoing cross-version compatibility issues trying to keep them in sync.

**Comment by gauravtiwari** [comment]
Fixed by #285
